src/main/kotlin/racingcar/domain/RaceBoard.kt

```diff
@@ -0,0 +1,14 @@
+package racingcar.domain
+
+import racingcar.domain.dto.Winner
+
+class RaceBoard(
+    val raceResultPerRounds: List<RaceResultPerRound>,
+) {
+    fun findWinners(): List<Winner> {
+        val lastRound =
+            raceResultPerRounds.lastOrNull()
+                ?: return emptyList()
+        return lastRound.findWinners()
```

Comment:
엘비스 연산자로 early return 하는 로직은 개인적으로 파악하기가 어렵더라구요 🥲
아래와 같이 변경해보는건 어떨까요? 😃 

```suggestion
        val lastRound = raceResultPerRounds.lastOrNull()
        return lastRound?.findWinners() ?: emptyList()
```

물론 상황에 따라서 이런 early return 구문을 활용해야하는 경우도 있지만, 저는 가급적 이런 식으로는 사용하지 않으려고하는 편이에요.

Reply:
조금 더 가독성이 좋아진것같아요! 반영하였습니다🙂